src/components/file-upload/FileUpload.js

```diff
@@ -63,26 +63,28 @@ export default class FileUpload extends PureComponent {
     const {
       disabled,
       children,
-      dropZoneStyle,
       uploads,
       onNotify,
+      onReject,
+      onUpload,
       onHideNotification,
+      ...rest
     } = this.props
+
     const { isDragging } = this.state
 
     return (
       <Dropzone
         disableClick
         disablePreview
-        disabled={disabled}
-        style={dropZoneStyle}
         onDrop={this.onDragDone}
         onDragStart={this.onDragStart}
         onDragEnter={this.onDragEnter}
         onDragLeave={this.onDragDone}
         onDropAccepted={this.onAccept}
         onDropRejected={this.onReject}
         maxSize={maxSize}
+        {...rest}
```

Comment:
Let's try to use explicit props instead of spreading. It quickly becomes a trap for unexpected behaviour.

Reply:
Yep sounds good! I think `style`, `disabled` & `className` is what we would need here.